druid-shell/src/gtk/mod.rs

```diff
@@ -87,6 +87,8 @@ pub struct WindowBuilder {
     handler: Option<Box<dyn WinHandler>>,
     title: String,
     menu: Option<menu::Menu>,
+    width: f32,
```

Comment:
I would use the `kurbo::Size` type for this; it's the type we use to represent sizes elsewhere, and it's the type we will use in the public api in druid. 👌